src/facebook_to_engagement_db/facebook_to_engagement_db.py

```diff
@@ -142,9 +148,11 @@ def _fetch_and_sync_facebook_to_engagement_db(google_cloud_credentials_file_path
     for dataset in facebook_source.datasets:
         # Download and sync all the comments on all the posts in this dataset.
         for post_id in _get_facebook_post_ids(facebook, facebook_source.page_id, search=dataset.search):
+            latest_comment_timestamp = None if cache is None else cache.get_latest_comment_timestamp(post_id)
             post_comments = facebook.get_all_comments_on_post(post_id,
-                        fields=["from{id}", "parent", "attachments", "created_time", "message"]
-                    )
+                                                              fields=["from{id}", "parent", "attachments",
+                                                                      "created_time", "message"],
+                                                              )
```

Comment:
Is there an auto-format rule somewhere that is causing changes like this?

In general, I think it is better to indent args like this by 1 tab rather than at the brackets, in order to stop all the arguments getting crammed in the right edge of the screen
e.g. 
```
post_comments = get_all_comment_on_post(
    post_id, ["parent", "attachments", "created_time", "message"]
)
```
is slightly more easily read than:
```
post_comments = get_all_comments_on_post(post_id,
                                         ["parent",
                                          "attachments",
                                          "created_time",
                                         "message"]
                                         )
```